src/modules/auth/controllers/meemoo.controller.ts

```diff
@@ -164,6 +164,7 @@ export class MeemooController {
 					time: new Date().toISOString(),
 					data: {
 						idp: Idp.MEEMOO,
+						user_group: archiefUser.groupId,
```

Comment:
it's better to use groupName instead of groupId since those uuid don't mean anything outside of the hetarchief database. So the groupName (eg: CP_ADMIN) will be more meaningful